src/risk_distributions/risk_distributions.py

```diff
@@ -455,7 +455,7 @@ def pdf(self, x: Union[pd.Series, np.ndarray, float, int]) -> Union[pd.Series, n
         p.loc[computable] = 0
         for name, parameters in self.parameters.items():
             w = weights.loc[computable, name]
-            p += w * self.distribution_map[name](parameters=parameters).pdf(x.loc[computable])
+            p += w * self.distribution_map[name](parameters=parameters.loc[computable]).pdf(x.loc[computable])
```

Comment:
We only want to filter down to computable if len(parameters) != 1.  Otherwise we break the interactive use case.

Reply:
good catch. just fixed